active-fedora.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "active-triples", '~> 0.11.0'
   s.add_dependency "deprecation"
   s.add_dependency "ldp", '~> 0.6.0'
+  s.add_dependency "ruby-progressbar", '~> 1.0'
```

Comment:
oh man, this is really a pain to test for, although I see your point.  I'll think about a way to test it, not sure when I'll get to it. If you have any advice or examples of testing output to stderr/stdout like this, feel free to share.